editor/plugins/script_editor_plugin.cpp

```diff
@@ -520,6 +520,8 @@ void ScriptEditor::_update_modified_scripts_for_external_editor(Ref<Script> p_fo
 	if (!bool(EditorSettings::get_singleton()->get("external_editor/use_external_editor")))
 		return;
 
+	ERR_FAIL_COND(!get_tree());
```

Comment:
Is this really sufficient to fix this issue? Or did you forget to commit other files changed in 6687484?

Reply:
jup, the other changes are related to the "better user experience" part of the commit.
It crashed at line 527 (now) as get_tree() returned null.